### PR TITLE
replace type(''), type(b''), and type(0) with str, bytes, and int

### LINKED ISSUE #498: replace `type('')`, `type(b'')`, and `type(0)` with `str`, `bytes`, and `int`
it was relatively common when porting to use these in place of the switched-on types

### PR DESCRIPTION
**Pull request by MarcoGorelli**
closes #498

### CODE REVIEW

File: t.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+type(True)
```
Comment (asottile) [inline t.py:1]: he he he

Comment (MarcoGorelli) [reply t.py]: caught me :flushed:

**Comment by MarcoGorelli** [comment]
this'll do the rewrite even if the string isn't empty, e.g. `type("foo")` - is that OK or does it need excluding?

**Comment by asottile** [comment]
> 
> 
> this'll do the rewrite even if the string isn't empty, e.g. `type("foo")` - is that OK or does it need excluding?

yep! that's fine

File: README.md [hunk 2]
```diff
@@ -458,6 +458,23 @@ Availability:
...
+-type(0.)
++float
+```
+
+
+### Rewrite `type` of primitive
+
+Availability:
+- `--py3-plus` is passed on the commandline.
+
```
Comment (asottile) [inline README.md:477]: looks like the header is out of place

Comment (MarcoGorelli) [reply README.md:473]: indeed, thanks!

**Review by asottile — APPROVED** [review]

**Pull request merged**